tests/integration/base.py

```diff
@@ -143,43 +145,70 @@ class DBTIntegrationTest(unittest.TestCase):
     prefix = f'test{_runtime}{_randint:04}'
     setup_alternate_db = False
 
-    def databricks_cluster_profile(self):
-        return {
+    def _build_databricks_cluster_profile(
+        self,
+        http_path: str,
+        catalog: Optional[str] = None,
+        session_properties: Dict[str, str] = None,
+    ):
+        profile = {
             'config': {
                 'send_anonymous_usage_stats': False
             },
             'test': {
                 'outputs': {
-                    'dbsql': {
+                    'dev': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
-                        'http_path': os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH'),
+                        'http_path': http_path,
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
                 },
-                'target': 'dbsql'
+                'target': 'dev'
             }
         }
+        if catalog is not None:
+            # TODO: catalog should be set as 'catalog' or 'database'
+            #       instead of using 'session_properties'
+            # profile['test']['outputs']['dev']['catalog'] = catalog
+            if session_properties is not None:
+                session_properties['databricks.catalog'] = catalog
```

Comment:
What if the `session_properties` already have the 'databricks.catalog' field? How about throwing an exception for now if the catalog field is specified? For example:
```
if catalog is not None:
  throw Exception(f"Specifying catalog name is currently not supported. Please use session property 'databricks.catalog' = {catalog} instead.")
```

Reply:
This is a temporary implementation.
The `catalog` should be in the profile as `catalog` or `database` as in the TODO comment.
I don't think we need to do it here because we don't have a profile with `session_properties` specified so far.
(We should rather remove the argument and add it later when it's needed?)

> What if the `session_properties` already have the 'databricks.catalog' field?

That's a good point. We should handle the case when we have the actual implementation. cc @superdupershant

Reply:
Sounds good!